src/code/providers/google-drive-provider.js

```diff
@@ -330,22 +330,22 @@ class GoogleDriveProvider extends ProviderInterface {
             }
           })
         }
-        var download = (url, fallback) => {
+        var download = (url) => {
```

Comment:
On line 319 we ask for the `file.downloadUrl`  

A response in the bug report ( https://issuetracker.google.com/issues/149891169#comment7 ), suggests we should be asking for webContentLink instead. I am not sure if we tried that, or if it would work in our case, but it would be great if we could use the API cleanly.

Reply:
The webContentLink doesn't have the right mimeType or CORs headers to load the file into the app.  Its really a download link to save the file from your browser to your local machine.